modules/core/src/main/java/org/apache/fluo/core/client/FluoAdminImpl.java

```diff
@@ -221,23 +225,46 @@ private void initialize(Connector conn) throws Exception {
         CuratorUtil.NodeExistsPolicy.FAIL);
     CuratorUtil.putData(curator, ZookeeperPath.ORACLE_GC_TIMESTAMP, new byte[] {'0'},
         CuratorUtil.NodeExistsPolicy.FAIL);
+  }
 
-    conn.tableOperations().create(config.getAccumuloTable(), false);
-    Map<String, Set<Text>> groups = new HashMap<>();
-    groups.put("notify", Collections.singleton(ByteUtil.toText(ColumnConstants.NOTIFY_CF)));
-    conn.tableOperations().setLocalityGroups(config.getAccumuloTable(), groups);
-
-    IteratorSetting gcIter = new IteratorSetting(10, "gc", GarbageCollectionIterator.class);
-    GarbageCollectionIterator.setZookeepers(gcIter, config.getAppZookeepers());
+  private String encodeColumnFamily(Bytes cf) {
```

Comment:
I think this code was copied from Accumulo?  We should open issues in Accumulo to improve the ability to set iterators and locality groups on new table config.  I'll open those issues.

Reply:
Modified a bit, but yeah, mostly copied. Agreed on better API in Accumulo for this.

Reply:
https://issues.apache.org/jira/browse/ACCUMULO-4732